app/services/apply_api/retrieve_applications.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module ApplyApi
+  class RetrieveApplications
+    include ServicePattern
+
+    SUCCESS = 200
+
+    RECRUITMENT_CYCLE_YEAR = 2021
```

Comment:
Can we move it to settings please

FYI
https://github.com/DFE-Digital/teacher-training-api/blob/90db22edd210569fcbc98251f80fb2bcde9a3be4/config/settings.yml#L31